### ISSUE TITLE
feedback forum has moved

### ISSUE THREAD

**Issue by koconnor719** [department-of-veterans-affairs/vets-website#1305]
the feedback forum has changed randomly on 3/3?; it's now on the left side of page and you can no longer click on "feedback forum and see the entire list of comments.  it appears this has reverted back to old version of how we had this?  checked this on safari and chrome if that matters.  can we go back to the way it was before?   

@emilyville @awong-dev 

![screen shot 2016-03-03 at 9 49 03 pm](https://cloud.githubusercontent.com/assets/13770771/13516676/654f7d20-e18a-11e5-981d-18b627bb21b3.png)

**Comment by ayaleloehr** [comment]
cc @dannychapman since I know he was working on the feedback section at one point.

**Comment by emilyville** [comment]
this was a deployment issue and it was fixed. closing.

**Comment by koconnor719** [comment]
Ty!!!!



Sent with Good (www.good.com)


-----Original Message-----

**Comment by koconnor719** [comment]
this is not closed.  the feedback forum appears differently on throughout the site. i think it should be consistent if that is in line w/ UX best practice? we had it formatted like this based on user feedback before--users want to be able to see all the feedback --and not have to go thru entering a comment first. 

<img width="648" alt="screen shot 2016-03-14 at 10 17 55 am" src="https://cloud.githubusercontent.com/assets/13770771/13747396/700f7386-e9ce-11e5-9419-cfd0e653bafc.png">

**Comment by koconnor719** [comment]
reassigning to tiffany as i believe she is focusing on header and footer; @webinista pls. fix the feedback forum on homepage when you have time. should be the "feedback forum" layout and feature (like the one from the VEC page footer.  the homepage is currently incorrect.  ty.

**Comment by webinista** [comment]
@koconnor719 We've changed this in #1405. It's been closed as an item, but I doesn't look like it's been deployed to production yet.

**Comment by awong-dev** [comment]
Deploy is waiting on me. Let me do that now.

**Comment by koconnor719** [comment]
Thank you
Kelly O�Connor
VA Digital Service
kelly.o�connor@va.gov
202-299-4436

**Comment by koconnor719** [comment]
@webinista this is not closed; i've reopened 1405.  pls. let me know if you need any add'l info from me.  I'm not sure what's happening w/ this change but there are inconsistencies across the site.

**Comment by webinista** [comment]
My suspicion is that the change hasn't been folded into the current version of the Veterans Employment Center. 

@emilyville @awong-dev Can you help out here? The footer changes are available in the `jobs-employment-v2` branch, but they haven't been pushed to the site just yet. Are we holding off on pushing VEC changes until version 2 is ready?

**Comment by awong-dev** [comment]
@webinista if you merged to master...it should have deployed?  Do you have a link to the PR with the master merge of this change?

**Comment by koconnor719** [comment]
this is NOT urgent at all.  i just want to make sure the correct version of the feedback forum is pushed to production.  it is not the one currently on the homepage--with 2 links?  i don't know where that came from.  the correct version, based on my understanding of the only Veteran feedback we have on this, is the version that's currently on the VEC page.  does that make sense?  @emilyville @webinista

**Comment by awong-dev** [comment]
It's been open for to long...let's figure out how to fix it...

**Comment by koconnor719** [comment]
@awong-dev talked to emily and we figured it out!!! i'm closing this and have confirmed!  ty:)
